### PR TITLE
Help beautify

### PR DESCRIPTION
**Pull request by Koramund**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -34,6 +36,8 @@
 @bot.event
 async def on_ready():
 	log.info(f'Logged in as {bot.user}')
+	ending_note = "Powered by " + str(bot.user)[:-5] + "\nFor command {help.clean_prefix}{help.invoked_with}"
```
Comment (juliavdkris) [inline src/main.py]: Bit cleaner, and won't explode if Discord for some reason switches to 5 char tags or something

```suggestion
	ending_note = f'Powered by {bot.user.name}\nFor command {{help.clean_prefix}}{{help.invoked_with}}'
```

**Review by juliavdkris — COMMENTED** [review]
Also: add `discord-pretty-help` to the `requirements.txt`, otherwise the image can't be built lmao

File: src/main.py (hunk 1 above)
Comment (Koramund) [reply src/main.py]: Can't use f String since it uses help.clean_prefix somewhere else, updated it to use split and join instead.

Comment (Koramund) [reply src/main.py]: and force pushed to use bot.user.name, man I'm bad

File: src/main.py [hunk 2]
```diff
@@ -34,6 +36,8 @@
 @bot.event
 async def on_ready():
 	log.info(f'Logged in as {bot.user}')
+	ending_note = 'Powered by ' + bot.user.name + '\nFor command {help.clean_prefix}{help.invoked_with}'
```
Comment (juliavdkris) [inline src/main.py]: Fixed my f-string suggestion to work with the string formatting that the help library applies later

```suggestion
	ending_note = f'Powered by {bot.user.name}\nFor command {{help.clean_prefix}}{{help.invoked_with}}'
```

**Review by juliavdkris — APPROVED** [review]

**Pull request merged**